src/Microsoft.ML.Recommender/MatrixFactorizationTrainer.cs

```diff
@@ -89,8 +89,19 @@ namespace Microsoft.ML.Trainers
     public sealed class MatrixFactorizationTrainer : TrainerBase<MatrixFactorizationPredictor>,
         IEstimator<MatrixFactorizationPredictionTransformer>
     {
+        public enum LibMFLossFunctionType { SquareLossRegression = 0, SquareLossOneClass = 12 };
```

Comment:
>LibMFLossFunctionType [](start = 20, length = 21)

this is nested enum, you don't have to be so specific, can you just call it LossFunction or LossFunctionType?
LibMF prefix is a bit scary.